hardhat.config.ts

```diff
@@ -16,7 +20,13 @@ require('./tasks/PrizeTierHistory')
 
 const optimizerEnabled = true
 
-module.exports = {
+interface HardhatUserConfigExtended extends HardhatUserConfig {
+  namedAccounts: any;
+  external: any;
+  dependencyCompiler: any;
+}
```

Comment:
You should not have to do this, `namedAccounts` should be defined in `HardhatUserConfig`.
Maybe bump dependencies version for hardhat to see if it fixes the problem?